test/test_notebook.py

```diff
@@ -83,6 +83,9 @@ def test_agentchat_MathChat(save=False):
 def test_oai_chatgpt_gpt4(save=False):
     run_notebook("oai_chatgpt_gpt4.ipynb", save=save)
 
+def test_hierarchy_flow_using_select_speaker(save=False):
```

Comment:
Could you please add a `@pytest.mark.skipif` decorator like other tests to run it only in one py version, 3.10 or 3.11?